website/addons/mendeley/model.py

```diff
@@ -14,11 +14,12 @@
 from .api import APISession
 
 
-def serialize_folder(name, account_id, parent_id=None, list_id=None):
+def serialize_folder(name, account_id, parent_id=None, list_id=None, id=None):
     retval = {
         'name': name,
         'provider_account_id': account_id,
         'provider_list_id': list_id,
+        'id': id
```

Comment:
@samchrisinger: This seems to be a duplication of `provider_list_id` - what is its purpose?